__tests__/functional/register.test.js

```diff
@@ -0,0 +1,125 @@
+import AppRunner from "./utils/AppRunner.js";
+import {
+    expectNoErrors,
+    expectAnyError,
+    expectAnyErrorMessageToBe,
+} from "./utils/matchers.js";
+
+const collectionsEmptiedOnCleanup = ["users"];
+let SUT = new AppRunner(collectionsEmptiedOnCleanup);
+
+const registerQuery = `mutation registerUser($username: String!, $password: String!, $email: String!) {
+							register(username: $username, password: $password, email: $email) {
+								user_id
+								username
+							}
+						}`;
+
+function createRegisterUserMsg() {
+    return {
+        query: registerQuery,
+        operationName: "registerUser",
+        variables: {
+            username: "test_user",
+            email: "grzesiekkasprzak", //currently this email is accepted!
+            password: "qwerty",
+        },
+    };
+}
```

Comment:
If, the code is buggy, it's good for the test to throw an error. The integration test should alert you to the problem. 
So this data should check to see if the code returns an error.

Reply:
None of these tests was designed to check validation of e-mails, so I used any valid e-mail at the moment of writing. I noticed that many strings are accepted, so I notified you and someone should fix it.
More tests (also with different e-mails) will be delivered in the future, after merging this review request, and they will detect such cases and hopefully fail.

Reply:
Latest commit changes email to valid so these tests won't fail after this bug is fixed.
I created a card on Trello (no 65) for adding tests specifically testing different emails. Of course they should pass, so task will require adding email validation on backend.